tools/scripts/brokenLinkChecker/brokenLinkChecker.json

```diff
@@ -71,13 +87,26 @@
     "http://www.usnews.com/news/stem-solutions/articles/2014/11/25/making-it-count-computer-science-spreads-as-graduation-requirement",
     "http://www.usnews.com/opinion/knowledge-bank/articles/2016-02-11/arkansas-and-washington-lead-efforts-to-expand-computer-science-education",
     "http://www.wsj.com/articles/why-coding-is-your-childs-key-to-unlocking-the-future-1430080118",
+    "http://www.youtube.com/watch?v=FC5FbmsH4fw",
+    "http://www.youtube.com/watch?v=MwLXrN0Yguk",
+    "http://www.youtube.com/watch?v=nKIu9yen5nc",
+    "http://www.youtube.com/watch?v=qYZF6oIZtfc",
+    "http://youtu.be/hfZrX5YDltU",
```

Comment:
Is it possible we should ignore all youtube URLs?

Reply:
That thought has crossed my mind.  It's certainly possible to have a broken youtube link (videos do get taken down) but the particular link checker tool we're using has a terrible time checking them accurately.  I'm not sure why, exactly.

I'll fast-follow with this change.